### PR TITLE
Changeintro

### PR DESCRIPTION
**Pull request by danheadforcode**
Added steps for the users to take so they know what to do.

### CODE REVIEW

File: src/containers/HomePage/IntroBlock.js [hunk 1]
```diff
@@ -23,15 +23,15 @@ class IntroBlock extends React.Component {
         </div>
         <div className="fl-m fl-l w-33-m w-33-l">
           <div className="pa4 pa4-m">
-            <h3 className="f3">We are seeking photos</h3>
+    <h3 className="f3">2.) We are seeking photos</h3>
```
Comment (nickeblewis) [inline src/containers/HomePage/IntroBlock.js]: Maybe I didn't explain things right but we didn't want to change this line.... please change it back to what it was, thanks. Step 2 will come later in another modification.

File: src/containers/HomePage/IntroBlock.js [hunk 2]
```diff
@@ -23,15 +23,15 @@ class IntroBlock extends React.Component {
...
             <p className="mid-gray f4 lh-copy measure-narrow">
               We are launching the site early by introducing our local photography campaign. We invite everyone to share their photgraphs of the town here on Farnborough Guide. We are especially interested in businesses around the town whom wish to share photographs of what they do, what they sell. help spread the word.
             </p>
           </div>
         </div>
         <div className="fl-m fl-l w-33-m w-33-l br b--light-gray">
           <div className="pa4 pa4-m">
-            <h3 className="f3">Sign-up</h3>
+    <h3 className="f3">1.) Sign-up</h3>
```
Comment (nickeblewis) [inline src/containers/HomePage/IntroBlock.js]: Almost right - Please take out the bracker, should just be 1. Sign up

File: src/containers/HomePage/IntroBlock.js (hunk 1 above)
Comment (danheadforcode) [reply src/containers/HomePage/IntroBlock.js]: okay i will do that now

**Review by nickeblewis — CHANGES_REQUESTED** [review]
See comments - sorry I may have confused you over the step 2 thing.

File: src/containers/HomePage/IntroBlock.js (hunk 2 above)
Comment (danheadforcode) [reply src/containers/HomePage/IntroBlock.js]: okay will do this also then do a push ?

**Review by nickeblewis — APPROVED** [review]
Great stuff! Done

**Pull request merged**